### PR TITLE
[opt](profile) Add new config profile_waiting_time_for_spill_s

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
In previous, we use auto_profile_shreshold to control how many time duration will ProfileManager waiting for profile collection after query has finished. The default value of auto_profile_shreshold is -1, makes profile need to be reported immediately after query finished, or it will be not complete. So introduce a new config in Config to solve the problem.

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
image

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -2768,6 +2768,10 @@ public class Config extends ConfigBase {
     @ConfField
     public static long spilled_profile_storage_limit_bytes = 1 * 1024 * 1024 * 1024; // 1GB
 
+    // Profile will be spilled to storage after query has finished for this time.
+    @ConfField
+    public static int profile_waiting_time_for_spill_s = 10;
+
```
Comment (yiguolei) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2774]: _secs

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run feut

**Review by yiguolei — DISMISSED** [review]

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 2]
```diff
@@ -2784,6 +2784,10 @@ public class Config extends ConfigBase {
     @ConfField
     public static long spilled_profile_storage_limit_bytes = 1 * 1024 * 1024 * 1024; // 1GB
 
+    // Profile will be spilled to storage after query has finished for this time.
+    @ConfField
```
Comment (morningman) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2788]: add description field

Comment (morningman) [reply fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2788]: use `_second` as suffix, like other config item's name

Comment (zhiqiang-hhhh) [reply fe/fe-common/src/main/java/org/apache/doris/common/Config.java]: done

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]

**Pull request merged**